src/starter-kit wiki page folder/Tutorials-FAQ.md

```diff
@@ -0,0 +1,11 @@
+## Example dataset permission error
+
+I'm getting this error when trying to download the NITRC dataset, what should I do?
+![](https://user-images.githubusercontent.com/22478219/45544974-28066100-b819-11e8-86ff-b2342c8917e5.png)
+
+Here are a list of steps you can follow to request access
+
+![](https://user-images.githubusercontent.com/12662110/45588087-08387100-b8dd-11e8-8658-d3c15c6dc92f.PNG)

```

Comment:
I think we will want to grad the pictures that are linked in this document and add them to the repo. But this can be done in another pull request.

Reply:
I downloaded all these pictures and uploaded them into a new folder inside src. 

Please feel free to check. Thank you so much.